repos/system_upgrade/el7toel8/actors/checkfstaboptions/actor.py

```diff
@@ -0,0 +1,76 @@
+from collections import defaultdict
+
+from leapp.actors import Actor
+from leapp.exceptions import StopActorExecutionError
+from leapp.models import StorageInfo
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+from leapp import reporting
+
+
+class CheckFstabOptions(Actor):
+    """
+    Check for mount options that are invalid under RHEL 8.
+
+    Several XFS mount options have been removed from the kernel between RHEL 7 and 8.
+    Their occurence will lead to an unbootable machine. If one of them is detected, the upgrade is inhibited.
+
+    The list of removed options is taken from the xfs(5) manual page on RHEL 8.
+    """
+
+    name = 'check_fstab_options'
+    consumes = (StorageInfo,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    REMOVED_XFS_OPTIONS = [
```

Comment:
why do you need this constant inside the class variable?

Reply:
Also, if you would decide to move it in the module scope check this https://stackoverflow.com/questions/33700927/tuples-vs-lists-for-module-level-constants-in-python disucssion. It is nice to understand when you might need tuple ot list as a constant